jsonnet/telemeter/client/kubernetes/kubernetes.libsonnet

```diff
@@ -0,0 +1,112 @@
+local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
+local credentialsSecret = 'telemeter-client';
+local credentialsVolumeName = 'credentials';
+local credentialsMountPath = '/etc/telemeter';
+local fromCAFile = '/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt';
+local fromTokenFile = '/var/run/secrets/kubernetes.io/serviceaccount/token';
+local metricsPort = 8080;
+
+{
+  _config+:: {
+    namespace: 'default',
+
+    telemeterClient+:: {
+      from: 'https://prometheus-k8s.openshift-monitoring.svc:9091',
+      to: '',
+    },
+
+    versions+:: {
+      telemeterClient: 'v3.11',
+    },
+
+    imageRepos+:: {
+      telemeterClient: 'openshift/telemeter',
+    },
+  },
+
+  telemeterClient+:: {
+    deployment:
+      local deployment = k.apps.v1beta2.deployment;
+      local container = k.apps.v1beta2.deployment.mixin.spec.template.spec.containersType;
+      local volume = k.apps.v1beta2.deployment.mixin.spec.template.spec.volumesType;
+      local containerPort = container.portsType;
+      local containerVolumeMount = container.volumeMountsType;
+      local containerEnv = container.envType;
+
+      local podLabels = { 'k8s-app': 'telemeter-client' };
+      local credentialsMount = containerVolumeMount.new(credentialsVolumeName, credentialsMountPath);
+      local credentialsVolume = volume.fromSecret(credentialsVolumeName, credentialsSecret);
+      local id = containerEnv.fromSecretRef('ID', credentialsSecret, 'id');
+
+      local telemeterClient =
+        container.new('telemeter-client', $._config.imageRepos.telemeterClient + ':' + $._config.versions.telemeterClient) +
+        container.withCommand([
+          '/usr/bin/telemeter-client',
+          '--id=$(ID)',
+          '--from=' + $._config.telemeterClient.from,
+          '--from-ca-file=' + fromCAFile,
+          '--from-token-file=' + fromTokenFile,
+          '--to=' + $._config.telemeterClient.to,
```

Comment:
ah, I see, this is coming from configuration :+1:

Reply:
yes, once we know the production telemeter URL we should use that as the default value in this libsonnet file